src/main/ionic/peers/src/pages/home/home.scss

```diff
@@ -21,7 +21,7 @@ page-home {
   }
 
   .title {
-    font-weight: bold;
+    font-weight:bold;
```

Comment:
why was this changed? it should have a space.

Reply:
This was modified before, and I changed it back